scripts/convert-typo.ts

```
@@ -0,0 +1,107 @@
+
+// 실사용 CSS 파일 경로
+const outPath = path.resolve('src/shared/styles/font-tokens.css');
+
+// 출력 디렉터리 보장
+fs.mkdirSync(path.dirname(outPath), { recursive: true });
+
+let cssOutput = `/*auto generated by convert-typo.ts*/\n@layer utilities {\n`;
+for (const [groupName, groupObj] of Object.entries(groups)) {
+  if (!groupObj) continue;
+
+  for (const [key, token] of Object.entries<any>(groupObj)) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)